### DISCUSSION TITLE
interaction positions check = FAIL (all interactions always happening on the same position)

### DISCUSSION THREAD

**Discussion by moferry96** [metarank/metarank#1104] [Q&A]
Hi.
When I run the validation command on my config and data files I get this failure:

> a.m.v.c.InteractionPositionValidation$ - interaction positions check = FAIL (all interactions always happening on the same position)

What are the causes of this error and how can one address this issue?

Thanks in advance.

**Comment by vgoloviznin** [comment]
Hey @mhfereydouni, as the error suggests, all the click events in your dataset contain clicks on the same position. This will definitely cause a position bias and will not produce a good personalization model. 

The best way is to get close-to-real dataset, so that you get good enough results.

**Reply by moferry96** [reply]
What does it mean by "clicks on the same position"?

**Reply by vgoloviznin** [reply]
It means that all the click events happened on the same position, e.g. position 1 of the results.
